Xamarin.Forms.Platform.UAP/ButtonRenderer.cs

```diff
@@ -290,5 +293,24 @@ void UpdatePadding()
 				Element.Padding.Bottom
 			);
 		}
+
+		bool _isDisposed;
+		protected override void Dispose(bool disposing)
+		{
+			if (_isDisposed)
+				return;
+			if (_button != null)
```

Comment:
This is missing the check for `disposing == true`; if this is called by the finalizer, it could potentially crash in weird ways.